ansible/roles/datadog/templates/kafka_consumer.yaml.j2

```diff
@@ -0,0 +1,35 @@
+init_config:
+#  Customize the ZooKeeper connection timeout here
+  zk_timeout: 5
+#  Customize the Kafka connection timeout here
+  kafka_timeout: 5
+
+instances:
+  - kafka_connect_str: localhost:9092  # Should always be deployed to broker so localhost is OK
+    zk_connect_str: "{{ groups['zookeeper'][0] }}:{{ zookeeper_client_port }}"
+    zk_prefix: /0.8
+    consumer_groups:
+      'form-processsor':
```

Comment:
the list looks good, though maintaining this here in duplicate with the code doesn't feel like a great long-term solution.

we don't really use groups at all yet. i think the way we would start using them is if we needed to have multiple pillows of the same type because we can't handle the throughput on a single one. also as you noted our partition setup is pretty hard coded for a single partition per topic. 

i don't think any of this blocks anything on this PR, just noting that there are some architectural issues with scaling out the environment if and when we need to do that